http-client/Network/HTTP/Client/Types.hs

```diff
@@ -715,6 +715,11 @@ data Response body = Response
     -- via @getOriginalRequest@ instead.
     --
     -- Since 0.7.8
+    , responseEarlyHints :: ResponseHeaders
+    -- ^ Early response headers sent by the server, as part of an HTTP
+    -- 103 Early Hints section.
+    --
+    -- Since 0.7.16
```

Comment:
In the other PR I asked for a version bump, but if you can make this the new version and add a changelog/cabal file change, that would be perfect.

Reply:
:+1:  This PR slightly improves on some aspects of the other one. I'll just close the other one.

Reply:
Done